examples/morpheusvm/actions/transfer_test.go

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package actions
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/ava-labs/hypersdk/chaintesting"
+	"github.com/ava-labs/hypersdk/codec"
+	"github.com/ava-labs/hypersdk/examples/morpheusvm/storage"
+	"github.com/ava-labs/hypersdk/state"
+	"github.com/ava-labs/hypersdk/tstate"
+)
+
+func TestTransferAction(t *testing.T) {
+	require := require.New(t)
+	ts := tstate.New(1)
+	emptyBalanceKey := storage.BalanceKey(codec.EmptyAddress)
+
+	tests := map[string]chaintesting.ActionTest{
+		"ZeroTransfer": {
+			Action: &Transfer{
+				To:    codec.EmptyAddress,
+				Value: 0,
+			},
+			ExpectedErr: ErrOutputValueZero,
+		},
```

Comment:
Could we explicitly set the `Actor` field of each `ActionTest` ?

It's currently accessed by these tests and using `codec.EmptyAddress` since it's not populated.

Reply:
Done !